apps/dg/resources/case_card.css

```diff
@@ -253,3 +253,23 @@ tr:nth-child(even) td.attr-cell {
     color: #333333;
 }
 
+$column-resize-handle-interaction-width: 10px;
+.column-resize-handle {
+    position: absolute;
+    margin-left: -$column-resize-handle-interaction-width / 2;
+    width: $column-resize-handle-interaction-width;
+    top: 0;
+    height: 100%;
+    cursor: col-resize;
+
+    .column-resize-handle-render {
+        position: absolute;
+        left: 0;
+        width: 50%;
+        height: 100%;
+
+        &.is-resizing {
+            border-right: 1px dotted black;
+        }
+    }
+}
```

Comment:
We never got in the habit of making use of the fact that SproutCore runs all of the CSS files it processes through SASS, so we can use variables and nesting. Now that I've used SASS on other projects I'm glad I don't have to live without it here.